solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -86,14 +86,26 @@ passwd root
 ```
 ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"<AWS instance public-ip>"
 ```
-- Clone cortx-re repository and switch to `solutions/kubernetes` directory
-```
-git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/community-deploy
-```
-- Execute `build-cortx.sh` script. This script will generate CORTX container images from `main` of CORTX components
-```
-time ./build-cortx.sh
-```
+- Clone cortx-re repository and switch to `solutions/community-deploy` directory
+    - If you want to build CORTX from `main` branch
```

Comment:
To make it simpler we can separate `git clone` and `cd` command. Something like - 

Clone cortx-re repository from required branch/tag 

`git clone https://github.com/Seagate/cortx-re -b <branch/tag>  `

If you do not provide any parameter then it will use `main` branch 

Switch to solutions/community-deploy directory

`cd $PWD/cortx-re/solutions/community-deploy`

Reply:
Done